crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -2212,6 +2212,8 @@ static const struct ECDSATestVector kECDSATestVectors[] = {
      AWSLC_APPROVED},
     {NID_secp224r1, &EVP_sha512, AWSLC_APPROVED, AWSLC_APPROVED,
      AWSLC_APPROVED},
+    {NID_secp224r1, &EVP_sha512_256, AWSLC_APPROVED, AWSLC_APPROVED,
```

Comment:
Would these tests detect if the signature that is generated is invalid?

Reply:
They would not, added some new changes that do so 👍